### PR TITLE
Feature/news overview

### PR DESCRIPTION
**Pull request by beneverard**
Updated the news section with the new design

### CODE REVIEW

File: assets/scss/non-modular/_blog.scss [hunk 1]
```diff
@@ -1,235 +1,178 @@
...
+	@include break;
+}
 
-		@include mq(1400) {
-			@include font-size(30);
-		}
+.blog__post-items,
+.news__post-items {
+    display: flex;
+	flex-wrap: wrap;
```
Comment (ConnorHolyday) [inline assets/scss/non-modular/_blog.scss]: Check indentation

File: partials/card/stripped.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+	<div class="card__title">
+
+		<h2 class="card__heading">
+			<a class="card__link" href="<?php the_permalink(); ?>"><?php the_title(); ?></a>
+		</h2>
+
+	</div>
+
+    <p class="card__meta">
```
Comment (ConnorHolyday) [inline partials/card/stripped.php]: Indentation

File: partials/update/panel.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<div class="update-panel">
+
+	<div class="update-panel__subscribe">
+
+		<h3 class="update-panel__subscribe-title">Subscribe to our newsletter</h3>
+		<p>Sign up to out monthly email where you can recieve updates on our current work</p>
```
Comment (ConnorHolyday) [inline partials/update/panel.php]: Does this need to be string translated for Spanish?

File: archive-news.php [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+		]);
+
+	?>
+
+	<div class="wrapper / news__container">
+
+		<div class="band band--thin">
+
+			<a href="<?php echo get_permalink(get_option('page_for_posts')); ?>" class="text-button button--icon">
+				<svg><use xlink:href="#icon-arrow-left"></svg><?php _e('Back to latest', 'ocp'); ?>
```
Comment (beneverard) [inline archive-news.php]: I'd say this probably isn't worth it. In HTML5 the trailing slash on a "self-closing element" is ignored anyway. I guess the real question here is consistency, and we should probably start to move towards no slashes throughout.

File: assets/scss/modular/_blog-filter.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+	z-index: 1;
+
+	li {
+		margin: 0;
+		padding: 0;
+	}
+
+	a {
+
+		color: currentColor;
```
Comment (beneverard) [inline assets/scss/modular/_blog-filter.scss]: 👍

File: assets/scss/modular/_card.scss [hunk 6]
```diff
@@ -685,3 +692,127 @@ category:
...
+	right: 0;
+	bottom: 0;
+	left: 0;
+	color: color('white');
+	background: linear-gradient(0deg, #272727 0%, rgba(15,15,15,0.46) 53.7%, rgba(74,74,74,0) 100%);
+	padding-top: 7.5%;
+}
+
+.card--featured .card__meta {
+	margin: 0 0 spacing(2) 0;
```
Comment (beneverard) [inline assets/scss/modular/_card.scss]: Good point, resolved.

File: assets/scss/non-modular/_blog.scss (hunk 1 above)
Comment (beneverard) [inline assets/scss/non-modular/_blog.scss]: Well spotted, fixed throughout repo.

File: partials/card/stripped.php (hunk 2 above)
Comment (beneverard) [inline partials/card/stripped.php]: Well spotted, fixed throughout repo.

File: partials/update/panel.php (hunk 3 above)
Comment (beneverard) [inline partials/update/panel.php]: Yes it will, thanks for spotting, resolved.

File: assets/scss/modular/_blog-filter.scss [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+	}
+
+	strong {
+		@include font('primary');
+	}
+
+}
+
+.blog-filter__options {
+	@include font('primary');
```
Comment (beneverard) [inline assets/scss/modular/_blog-filter.scss]: Hmmm, true, but I feel that it makes it less readable. Sure, with the font weights it's expected that the default weight is "normal" (in most instances), however, I'm not sure that applies in the same way to the font mixin.

**Pull request merged**